sphinx/templates/quickstart/conf.py_t

```diff
@@ -112,12 +112,16 @@ html_static_path = ['{{ dot }}static']
 #
 # This is required for the alabaster theme
 # refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
-html_sidebars = {
-    '**': [
-        'relations.html',  # needs 'show_related': True theme option to display
-        'searchbox.html',
-    ]
-}
+if html_theme == 'alabaster':
+    html_sidebars = {
+        '**': [
+            'about.html',
+            'navigation.html',
+            'relations.html',  # needs 'show_related': True theme option to display
```

Comment:
as this needs extra activation step for Alabaster, I think it should be commented out

Reply:
The whole point of the `if` is that it's *not commented out* so that it's active by default for a new project, since the *default* theme is Alabaster. When the user changes the theme, he will likely forget to remove this, which is what #3987 was about. The `if` means it won't apply even if the user doesn't notice this, which means his newly selected theme will work correctly.

Reply:
yes I understand that. But I think for Alabaster user it makes more sense to have ``'relations.html'`` commented out because it needs extra step as explained in comment. Having the whole line commented out would make it even clearer to user.